pkg/platform/model/buildplanner.go

```diff
@@ -100,29 +100,26 @@ func NewBuildPlannerModel(auth *authentication.Auth) *BuildPlanner {
 	}
 }
 
-func (bp *BuildPlanner) FetchBuildResult(commitID strfmt.UUID, owner, project string) (*BuildResult, error) {
+func (bp *BuildPlanner) FetchBuildResult(commitID strfmt.UUID) (*BuildResult, error) {
 	resp := &bpModel.BuildPlan{}
-	err := bp.client.Run(request.BuildPlan(owner, project, commitID.String()), resp)
+	err := bp.client.Run(request.BuildPlan(commitID.String()), resp)
```

Comment:
This looks wrong. We should always provide the project owner and name. According to this ticket the limitation should be addressed:

[PB-4359](https://activestatef.atlassian.net/browse/PB-4359)

Reply:
That ticket is describing the `stageCommit` mutation. This function is for fetching the results of a build. We are now using the new top-level `commit` query which only requires a commit ID.

In this PR, I've changed how we fetch the build plan so that we only ever use the commit ID. This allows us to use the same function for both local and remote commits. Do we want to fetch the build result using the `project` query when the commit is not local?

Here's the Build Planners schema for reference: https://github.com/ActiveState/TheHomeRepot/blob/master/service/buildplanner/buildplan.graphql

Reply:
That still doesn't feel right though. How can we grab the build result without project info considering it needs the project info in order to resolve private ingredients?